tests/templates/pvc-nfs.yaml

```diff
@@ -0,0 +1,75 @@
+# PV for NFS Provisioner Pod's PVC, i.e. underlying infra.
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: data-nfs-server-provisioner-0
+spec:
+  capacity:
+    storage: 5Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: /var/snap/microk8s/common/nfs-storage
+  claimRef:
+    namespace: nfs-server-provisioner
+    name: data-nfs-server-provisioner-0
```

Comment:
@berkayoz, please see `--set persistence.storageClass='-'` in `enable` script which disables dynamic provisioning of the underlying PV.
```
  ## Persistent Volume Storage Class
  ## If defined, storageClassName: <storageClass>
  ## If set to "-", storageClassName: "", which disables dynamic provisioning
  ## If undefined (the default) or set to null, no storageClassName spec is
  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
  ##   GKE, AWS & OpenStack)
  ##
  # storageClass: "-"
```
Dynamic provisioning of the underlying PV is normally ok, however I decided to mount nfs-provisioner's PV manually to the microk8s-native directory structure:  `path: /var/snap/microk8s/common/nfs-storage` to keep things clean and readable.

Does your suggested change still allow to use `nfs` StorageClass by further PVCs without any problems?

Reply:
Hey there, thank you for your time and interest.

It seems like what you mention about mounting manually is already done in `enable` which can be seen [here](https://github.com/canonical/microk8s-community-addons/blob/beaa6d3c39976c8617e32b4caeebd7e9e1fa7017/addons/nfs/enable#L56-L61). 
The resource definition in the suggested change is only used in testing and seems like unnecessary since we already call `enable` in the first place that would create this. 

I ran the tests with my suggested changes and everything seems to work as expected.